apps/sms/views/conversation/style/recipients.css

```diff
@@ -0,0 +1,198 @@
+#messages-to-field {
```

Comment:
We'll need dedicated class for the "recipients" component so that it isn't coupled to Composer view - like ```<div id="messages-to-field" class="recipients-container">```, but not in this patch